src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/uat/common/AbstractArgumentsProvider.java

```diff
@@ -0,0 +1,61 @@
+package uk.nhs.digital.nhsconnect.lab.results.uat.common;
+
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+import org.springframework.core.io.support.ResourcePatternResolver;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+abstract class AbstractArgumentsProvider implements ArgumentsProvider {
```

Comment:
Is this needed?

Reply:
No it isn't being used anywhere so I've removed it